testsuite/features/step_definitions/command_steps.rb

```diff
@@ -66,7 +67,8 @@
     # TODO: move that internal resource to some other external location
     STDERR.puts 'Sanity check not implemented, move resource to external network first'
   else
-    url = 'https://registry.mgr.suse.de:443'
+    registry_uri = ENV['REGISTRY_URI']
+    url = "https://#{registry_uri}"
```

Comment:
why not make `https://` part of the variable's content?

Reply:
It does not work in docker. That part is used directly for tagging and things like that.
https:// would break it.